### PR TITLE
[FEAT] Sensor events using template structure

### PR DESCRIPTION
**Pull request by sindhu-106**
Signed-off-by: sindhuja <sindhuja.gajula@intel.com>

### CODE REVIEW

File: docker-deploy/eventsim-config/EventSim.json [hunk 1]
```diff
@@ -13,25 +13,74 @@
...
+    "timezone": "GMT"
   },
   "network-config" : {
       "network" : "sse",
       "sse": {
           "server-address": "api-gw-service-nmn.local" ,
           "server-port": "8080" ,
           "urls": {
-            "/v1/stream/telemetry": [
+            "/v1/stream/cray-dmtf-resource-event": [
```
Comment (paul-amonson) [inline docker-deploy/eventsim-config/EventSim.json:25]: This breaks the "works out of the box" for the eventsim-config. You need to make sure the urls for adapters and eventsim in docker-deploy/eventsim-config are the same.

File: docker-deploy/eventsim-config/EventSim.json [hunk 2]
```diff
@@ -13,25 +13,74 @@
...
+              "events"
+            ] ,
+            "/v1/stream/cray-node": [
               "telemetry"
             ] ,
-            "/apis/smd/hsm/v1/Subscriptions/SCN": [
-              "stateChanges"
+            "/v1/stream/cray-logs-clusterstor": [
```
Comment (nbwhite) [inline docker-deploy/eventsim-config/EventSim.json:31]: Make sure to make the same change on the Provisioning Provider json and Inventory

File: docker-deploy/eventsim-config/EventSim.json [hunk 3]
```diff
@@ -13,25 +13,35 @@
...
+              "energy"
             ] ,
-            "/apis/sma-telemetry/v1/stream/cray-fabric-perf-telemetry": [
+            "/apis/sma-telemetry/apis/sma-telemetry/v1/stream/cray-fabric-perf-telemetry": [
               "fabric"
+            ] ,
+            "/apis/sma-telemetry/v1/stream/cray-telemetry-temperature": [
+              "telemetry"
+            ] ,
+            "/streams/nodeBootEvents": [
```
Comment (paul-amonson) [inline docker-deploy/eventsim-config/EventSim.json:43]: This is NOT a blocking comment. The state changes will also need the same updating of URLs as well as the providers for compatibility with the in house system software.

File: docker-deploy/eventsim-config/ProviderMonitoringNetworkForeignBus.json [hunk 4]
```diff
@@ -24,7 +45,10 @@
   "adapterProfiles": {
     "environmental": {
       "networkStreamsRef": [
-        "telemetry"
+        "energyTelemetry",
+        "powerTelemetry",
+        "temperatureTelemetry",
+        "voltageTelemetry"
```
Comment (paul-amonson) [inline docker-deploy/eventsim-config/ProviderMonitoringNetworkForeignBus.json:51]: Good!

**Pull request merged**